### PR TITLE
Fz12 fc 1

### PR DESCRIPTION
**Pull request by Tarnoruda**
nowa strona + zaimplementowana gra

### CODE REVIEW

**Review by piotr-andrzej-bochenek — APPROVED** [review]
Approved, but renderLife function is in it's old version (wrongly injecting innerHTML and appending children), it's much improved in PullRequest #29, awaiting for a review.

File: game/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (krichert) [inline game/.vscode/settings.json:2]: we do not need this in repository

File: game/game.css [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    min-width: 50px;
+    min-height: 50px;
+    background-color: gray; 
+}
+.map--floor{
+    background-image: url(tiles/floor.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    /* position: relative; */
```
Comment (krichert) [inline game/game.css:33]: left comment

File: game/game.css [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+    background-size: cover;
+}
+.map--floor--sewer{
+    background-image: url(tiles/floor3.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+}
+
+.map_wall--image{
```
Comment (krichert) [inline game/game.css:54]: once it is map_wall then map--floor, change it to map-wall and map-floor for example

File: main.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+});
+
+$('#link-extras').click(() => {
+    $('html, body').animate({
+        scrollTop: $('#extras').offset().top-20
+    }, 500);
+});
+
+// cookieInfo
+const cookies = document.querySelector(".cookies");
```
Comment (krichert) [inline main.js:33]: do we have a task in sprint about cookies?

**Review by krichert — CHANGES_REQUESTED** [review]

File: game/game.css (hunk 3 above)
Comment (piotr-andrzej-bochenek) [reply game/game.css:54]: changed to map--wall

File: main.js (hunk 4 above)
Comment (piotr-andrzej-bochenek) [reply main.js:33]: we don't have

**Review by karol-nogal — APPROVED** [review]

**Comment by Tarnoruda** [comment]
looks like done

**Review by karol-nogal — APPROVED** [review]

**Review by Tarnoruda — COMMENTED** [review]
28 / 122 re-viewed

**Review by krichert — APPROVED** [review]
Tylko ze względu na prezentację daje approve :)

**Pull request merged**